docs/src/prerequisites/index.md

```diff
@@ -17,3 +17,39 @@ you're excited about that work- you should reach out to the maintainers and get
 involved!
 
 [npm]: prerequisites/npm.html
+
+## Target wasm32-unknown-unknown
```

Comment:
i agree with your assessment that this might be best as it's own page- i think the title might better be phrased as "Non-rustup setups" this way, new folks who see this and are unsure, or just normal users with a rustup setup don't think they need to read this bit. 

I would make this its own page, replacing this section with a line that links to the page with this content saysing "Non-rustup setup? Learn how to configure for wasm-pack"- or something like this.

Reply:
Agreed with a separate page, and I think we'll want a disclaimer as well saying that it may not actually be possible to add the wasm32 target to all setups. The instructions below are correct for setups which match rustc nightlies/releases exactly (down to the precise commit being used to build the compiler). If anything is different though (aka if a distro has patches, or if it's not built the same way as Rust releases) then the downloaded wasm32 target will still be incompatible.

I think it's good to have the instructions below, but we probably still want a warning saying that it's not guaranteed to work.

Reply:
do we know of toolchains where it definitely will or wont work? it's always nice to list some examples.

Reply:
I'm not personally aware of either myself (I've only ever used rustup). It looks like the [DragonFly BSD toolchain](https://github.com/rustwasm/wasm-pack/issues/589#issuecomment-473934474) for sure isn't compatible due to it carrying local patches, but that's all I'd know of

Reply:
Yeah, I will move it to a separate page and add a disclaimer as well

Reply:
I have moved it to a separate page and I link to that from prerequisites. I also added a disclaimer. Feel free to comment on wording in the section as well. Thanks!